### PR TITLE
create English subject page

### PR DESCRIPTION
**Pull request by joegibb**

### CODE REVIEW

File: app/views/content/life-as-a-teacher/explore-subjects/english/_article.html.erb [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    </ul>
+    <p>In key stage 4 (14 to 16 years olds) you will help pupils demonstrate skills in:</p>
+    <ul>
+      <li>understanding and evaluating texts</li>
+      <li>analysing a writer’s choice of vocabulary, grammar and structure</li>
+      <li>writing accurately and fluently for pleasure</li>
+      <li>adapting writing style for different  purposes and audiences</li>
+      <li>analysing differences between written and spoken English</li>
+      <li>listening to the contributions of others</li>
+      <li>chemical and allied industries</li>
```
Comment (gemmadallmandfe) [inline app/views/content/life-as-a-teacher/explore-subjects/english/_article.html.erb:52]: should this line be removed?

File: app/views/content/shared/teaching/_bursary_only.html.erb [hunk 2]
```diff
@@ -1,4 +1,4 @@
 <div class="bursary_only">
-  <p><a href="/funding-and-support/scholarships-and-bursaries">Tax-free bursaries of <%= v "bursaries_postgraduate_#{@front_matter['subject'].gsub(/\s+/, '')}" %></a> are available for eligible trainee <%= @front_matter["subject"] %> teachers.</p>
+  <p><a href="/funding-and-support/scholarships-and-bursaries">Tax-free bursaries of <%= v "bursaries_postgraduate_#{@front_matter['subject'].gsub(/\s+/, '').sub(/^./, &:downcase)}" %></a> are available for eligible trainee <%= @front_matter["subject"] %> teachers.</p>
```
Comment (gemmadallmandfe) [inline app/views/content/shared/teaching/_bursary_only.html.erb:2]: Should probably get a quick tech review from @martyn-w on this?

File: app/views/content/life-as-a-teacher/explore-subjects/english/_article.html.erb [hunk 3]
```diff
@@ -0,0 +1,91 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Why teach English</h2>
+    <div class="inset">
+      <p>As an English teacher, you'll be a cornerstone in pupils' lives. From reading and exploring literature to writing and delivering debates, you'll cover essential skills needed to think critically and thrive beyond the classroom.</p>
```
Comment (Sarah-DfE) [inline app/views/content/life-as-a-teacher/explore-subjects/english/_article.html.erb:5]: I can't remember if I wrote this or you wrote this, but not sure I understand the first sentence? Feels more like being a cornerstone would be the role of a form tutor maybe?

File: app/views/content/life-as-a-teacher/explore-subjects/english/_article.html.erb [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+  <section class="col col-720">
+    <!-- Quote: Chris Farr -->
+    <%= render 'content/shared/quotes/quote_progression_chris' %>
+  </section>
+
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Shape your pupils' futures</h2>
+    <div class="inset">
+      <p>Teaching English opens the door for your pupils to pursue exciting careers in fields such as journalism, marketing and publishing. You may even inspire the next generation of teachers.</p>
```
Comment (Sarah-DfE) [inline app/views/content/life-as-a-teacher/explore-subjects/english/_article.html.erb:19]:
```suggestion
      <p>Teaching English opens the door for your pupils to pursue exciting careers in fields such as journalism, marketing, and publishing. You may even inspire the next generation of teachers.</p>
```

File: app/views/content/life-as-a-teacher/explore-subjects/english/_article.html.erb [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+  </section>
+
+  <section class="col col-720">
+    <!-- Quote: Holly, science teacher -->
+    <%= render 'content/shared/quotes/quote_english_laura' %>
+  </section>
+
+  <section class="col col-720">
+    <h2 class="heading--box-blue">What you'll teach</h2>
+    <p>You'll teach the <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-english-programmes-of-study/national-curriculum-in-england-english-programmes-of-study">national curriculum for English</a>.</p>
```
Comment (Sarah-DfE) [inline app/views/content/life-as-a-teacher/explore-subjects/english/_article.html.erb:31]: In the section below, key stage 3 the key stage is in brackets, but it's the other way round for key stage 4, with the ages in brackets. On other pages, the key stage is in brackets, so I'd maybe make this consistent too.

**Review by Sarah-DfE — COMMENTED** [review]
Just a few little comments @joegibb looking good tho!

File: app/views/content/life-as-a-teacher/explore-subjects/english/_article.html.erb [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+    <p>Skills you’ll cover when you teach 11 to 14 year olds (key stage 3) include:</p>
+    <ul>
+      <li>developing a love of reading increasingly difficult material</li>
+      <li>understanding how authors' choices can change meaning</li>
+      <li>summarising and organising material to support arguments</li>
+      <li>writing a range of stories, scripts, poetry and letters</li>
+      <li>planning, drafting and editing writing</li>
+      <li>using standard English confidently</li>
+      <li>giving short speeches and presentations</li>
+      <li>rehearsing and performing playscripts and poems</li>
```
Comment (Sarah-DfE) [inline app/views/content/life-as-a-teacher/explore-subjects/english/_article.html.erb:41]:
```suggestion
      <li>rehearsing and performing play scripts and poems</li>
```

Comment (Sarah-DfE) [reply app/views/content/life-as-a-teacher/explore-subjects/english/_article.html.erb:41]: Just changing for consistency as it's 2 words below

File: app/views/content/life-as-a-teacher/explore-subjects/english.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
+---
+title: Become an English teacher
+subject: English
+title_paragraph: |-
+  <p>
+  Teaching English is more than just reading and writing. It's opening  doors to new worlds and perspectives. You'll encourage a passion for language and a love of creativity, helping shape the voices of the future. </p>
```
Comment (Sarah-DfE) [inline app/views/content/life-as-a-teacher/explore-subjects/english.md:6]:
```suggestion
  Teaching English is more than just reading and writing. It's opening doors to new worlds and perspectives. You'll encourage a passion for language and a love of creativity, helping shape the voices of the future. </p>
```

File: app/views/content/life-as-a-teacher/explore-subjects/english.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+  <p>
+  Tax-free bursaries of $bursaries_postgraduate_english$ are available for eligible trainee English teachers.</p>
+description: |-
+    Find out how to become a English teacher, including what you'll teach and what funding is available to help you train.
```
Comment (Sarah-DfE) [inline app/views/content/life-as-a-teacher/explore-subjects/english.md:10]:
```suggestion
    Find out how to become an English teacher, including what you'll teach and what funding is available to help you train.
```

**Review by Sarah-DfE — COMMENTED** [review]
Made a few super small tweaks @joegibb

**Review by Sarah-DfE — APPROVED** [review]
Looks good to me!

**Pull request merged**